lib/cell/concept.rb

```diff
@@ -5,8 +5,23 @@ class Concept < Cell::ViewModel
 
     # TODO: this should be in Helper or something. this should be the only entry point from controller/view.
     class << self
-      def class_from_cell_name(name)
-        name.camelize.constantize
+      def class_from_cell_name(name, controller)
+        @cell_class ||= begin
+          supports_path = controller.supports_path?
+          routes  = controller.respond_to?(:_routes)  && controller._routes
+          helpers = controller.respond_to?(:_helpers) && controller._helpers
+
+          Class.new(name.camelize.constantize) do
```

Comment:
> believe it or not, but things broke everywhere and I had to fix it

I believe it. 

I also think I'm attempting a different approach as well. Including them during initialization is definitely a big problem. This is why, like Rails itself, the approach is to do it during the request, when we already have a guaranteed inclusion order, and consistency with existing views. A Cell has a parallel role to an individual ActionView::Base object, so I think, that in the case of Rails, it also makes sense to instantiate them in a similar manner, and possibly avoiding the same pitfalls that plagued Cells in the past.